include/mbedtls/cipher.h

```diff
@@ -405,17 +411,19 @@ void mbedtls_cipher_free( mbedtls_cipher_context_t *ctx );
  * In future versions, the caller will be required to call
  * mbedtls_cipher_init() on the structure first.
  */
-int mbedtls_cipher_setup( mbedtls_cipher_context_t *ctx, const mbedtls_cipher_info_t *cipher_info );
+int mbedtls_cipher_setup( mbedtls_cipher_context_t *ctx,
+                          const mbedtls_cipher_info_t *cipher_info );
 
 /**
  * \brief        This function returns the block size of the given cipher.
  *
- * \param ctx    The context of the cipher. Must be initialized.
+ * \param ctx    The context of the cipher. This must be initialized.
  *
  * \return       The size of the blocks of the cipher.
  * \return       0 if \p ctx has not been initialized.
  */
-static inline unsigned int mbedtls_cipher_get_block_size( const mbedtls_cipher_context_t *ctx )
+static inline unsigned int mbedtls_cipher_get_block_size(
+    const mbedtls_cipher_context_t *ctx )
 {
     if( NULL == ctx || NULL == ctx->cipher_info )
```

Comment:
This should be part of parameter validation.

Reply:
There is a technical issue regarding the `static` `inline` functions in this module. We agreed to only define the validating macros in the `.c` files. Thus for these tests to work, the definition would have to be moved to the header file. Do you think this is acceptable in this case? I don't think I see a particular issue with doing so, apart of introducing a new symbol in the public API.

Reply:
Oh, good point again. If we move the declaration, it must use the MBEDTLS namespace, and we would need to indicate through a comment like in `platform_util.h` that the macro is internal. Come to think of it, though, maybe it's easier to directly use `MBEDTLS_INTERNAL_VALIDATE[_RET]` here. Ping @mpg

Reply:
I agree that the best way is probably to use `MBEDTLS_INTERNAL_VALIDE[_RET]` here (and for inline functions in general).

Reply:
Thanks @mpg and @hanno-arm! I will do so.